docs/_rpc/server.md

```diff
@@ -3,8 +3,17 @@ title: JSON-RPC Server
 sort_key: A
 ---
 
-Geth supports all standard web3 JSON-RPC APIs. You can find documentation for
-these APIs on the [Ethereum Wiki JSON-RPC page][web3-rpc].
+Interacting with Geth requires sending requests to specific JSON_RPC API 
+endpoints. Geth supports all standard Web3 [JSON-RPC API][web3-rpc] endpoints. 
```

Comment:
```suggestion
methods. Geth supports all standard Web3 [JSON-RPC API][web3-rpc] endpoints. 
```

Reply:
Also I think "web3" is the wrong term here. Not sure if we have much of a better term though beyond "execution layer/client api". But IMO web3 has many different meanings.